pandas/core/indexes/multi.py

```diff
@@ -147,6 +147,11 @@ def _verify_integrity(self, labels=None, levels=None):
                                  " level  (%d). NOTE: this index is in an"
                                  " inconsistent state" % (i, label.max(),
                                                           len(level)))
+        for i, level in enumerate(levels):
+            if len(level) != len(set(level)):
+                raise ValueError("Level values must be unique: %s "
+                                 "on level %d" % ([value for value
```

Comment:
Can you use `.format` syntax instead of `%`?  I realize that other places within this file use `%` formatting, but there's an ongoing effort to transition all `%` formatting in the pandas codebase to `.format`, so might as well minimize the number of changes that will need to be made.

Reply:
Of course! I do prefer `.format` but decided to stick with `%` because of the other tests. Thank you for telling me, I'll keep it in mind in future contributions :)